SwiftLint/swiftlint+swiftUI.yml

```diff
@@ -0,0 +1,12 @@
+###
+# swiftlint+swiftUI.yml
+# For a swiftUI projects only
+#
+# This list of disabled rules gets appended to the swiftlint.yml when it gets copied into the root project directory.
+# You can add more "disabled_rules" here.
+
+# rule identifiers to exclude from running
+disabled_rules:
+  - multiple_closures_with_trailing_closure
+  - missing_closure_name
+  - missing_closure_datatype
```

Comment:
missing_closure_datatype: is there situations where it cannot work with swiftUI ?
IMO it should be that way:
- either you use $0, no need for type
- or you declare the variable, then we need the type
-> Anyway Xcode is autocompleting the type